src/lib.rs

```diff
@@ -27,17 +27,21 @@ pub mod mock_flash;
 /// Many flashes have 4-byte or 1-byte words.
 const MAX_WORD_SIZE: usize = 32;
 
+/// Align the given buffer to the word size of the flash.
+#[repr(align(32))]
+struct AlignedBuf([u8; MAX_WORD_SIZE]);
```

Comment:
32 is a bit much I think. If nordic needs 4, then I'm ok with that. Just make it 4 then.
```suggestion
#[repr(align(4))]
struct AlignedBuf([u8; MAX_WORD_SIZE]);
```